plugins/view-transitions/css/view-transition-animation-slide.css

```diff
@@ -10,6 +10,12 @@
 	inherits: false;
 }
 
+@property --plvt-view-transition-duration {
```

Comment:
I realize I suggested this name before, but apologies, I think this would be better as `--plvt-view-transition-animation-duration`, for parity with the option name and the CSS property. Can you please update this throughout the PR?